src/App.jsx

```diff
@@ -1,107 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(({ id }) => id === userId);
+}
+
+function getCommentsForPost(idForPost) {
+  return commentsFromServer
+    .filter(({ postId }) => postId === idForPost);
+}
```

Comment:
will be good in future move these functions into separated file or folder, like `helpers`, also `commentsFromServer.filter(...)` can be as single line

Reply:
this is a linter request to move the method to a new line